### PR TITLE
Add documentation comments for methods used to run 'fs ls' in Alluxio

### PR DESCRIPTION
**Pull request by abenaru-zz**
# Pull Request Merge Checklist
           
Task backlog identifier related to this PR: [277](https://github.com/s1mbi0se/simbiose_knowledge_base/issues/277)

---

## As a DEVELOPER, I confirm that this PR **doesn't contain**:
- [x] Code lines with syntax errors
- [x] Methods and variables names that aren't clear or doesn't mean anything
- [x] Code that is out of the standard (Example: **PEP-8** for Python code)
- [x] Files with **imports** errors or without any utility
- [x] **TODOs** left behind
- [x] Tests that are being skipped
- [x] Methods placed on incorrect files or folders (Example: methods that are local, but should be placed on /utils)
- [x] Functions, methods or API responses that are publicly returning sensitive information

## As a DEVELOPER, I confirm that I:
- [x] Followed the Simbiose's standard for branch and commit naming
- [x] Concluded the task and achieved the goals according to the task planning I defined before starting to code
- [x] Added all necessary database changes to the [SQL Migrations](https://github.com/s1mbi0se/sql-migrations)
- [x] Created tests for every new flow and method on this Pull Request
- [x] Added the necessary DEBUG logs
- [x] Checked my code using the Simbiose's Corner Cases Guide
- [x] Merged the **develop** branch to this PR/branch before sending the PR for review
- [x] Solved all possible merge conflicts before sending the PR for review
- [x] Corrected all the problems and code smells identified by the quality tools
- [x] Manually executed the necessary tests locally to make sure my changes didn't break anything
- [x] All unitary and integration tests ran by the CI system are passing
- [x] Check the performance of my changes or additions and they didn't make the system slower (I profiled the code)
- [x] Updated the necessary documentation (Example: OpenAPI, Docstrings, Metadata Database Modeling)
- [x] Made the self-review of the code before sending the PR for review
- [x] Sent this PR for a peer-review
- [x] Made all the corrections appointed during the peer-review
- [x] Confirmed that this PR merge will not cause other systems or dependencies to stop working (Example: ShannonDB Python Client and ShannonDB)
- [x] Checked all GitHub Actions related to this PR that can block my merge

---  

## As the TEAM MANAGER, I confirm that I:
- [x] Checked that the code was developed following the best practices for the language, with the best possible readability and using the less possible quantity of code lines and complexity. 
- [x] Checked if the Pull Request and commit messages follow the Simbiose’s defined standard 
- [x] Checked if there is no possible merge conflicts before reviewing the PR
- [x] Checked if the purpose of the task was reached with this PR
- [x] Checked that the code was developed following the best practices for the language, with the best possible readability and using the less possible quantity of code lines and complexity. 
- [x] Checked if all related PRs were created when other components were affected by this task (e.g. database changes on [SQL Migrations](https://github.com/s1mbi0se/sql-migrations))
- [x] Checked that were created unit, integration and chaos tests for every new flow and method on this Pull Request
- [x] Checked that no sensible information (e.g. users security data, internal security data, etc) was exposed by this PR
- [x] Checked that code coverage for this PR is higher than 80%.
- [x] Checked that there are no problems and code smells identified by the quality and security tools
- [x] Checked that all files were created on the proper folders according to the Simbiose’s defined standard
- [x] Checked that all classes, methods and variables on this PR follow the Simbiose’s pattern for this programming language
- [x] Checked that all unitary and integration tests ran by the CI system are passing
- [x] Checked that there are no performance issues created by these changes or additions
- [x] Checked that all documentation is up to date according to this change (Example: OpenAPI, Docstrings, Metadata Database Modeling)
- [x] Checked that this PR merge will not cause other systems or dependencies to stop working (Example: ShannonDB Python Client and ShannonDB)
- [x] Checked all GitHub Actions related to this PR that can block the merge

---  

## As the QUALITY RESPONSIBLE, I confirm that I:
- [x] Checked if the Pull Request and commit messages follow the Simbiose’s defined standard 
- [x] Checked if there is no possible merge conflicts before reviewing the PR
- [x] Checked that were created unit, integration and chaos tests for every new flow and method on this Pull Request
- [x] Checked that no sensible information (e.g. users security data, internal security data, etc) was exposed by this PR
- [x] Checked that code coverage for this PR is higher than 80%.
- [x] Checked that there are no problems and code smells identified by the quality and security tools
- [x] Checked that all files were created on the proper folders according to the Simbiose’s defined standard
- [x] Checked that all classes, methods and variables on this PR follow the Simbiose’s code standard for this programming language
- [x] Checked that all unitary and integration tests ran by the CI system are passing
- [x] Checked that there are no performance issues created by these changes or additions
- [x] Checked that all documentation is up to date according to this change (Example: OpenAPI, Docstrings, Metadata Database Modeling)
- [x] Checked that this PR merge will not cause other systems or dependencies to stop working (Example: ShannonDB Python Client and ShannonDB)
- [x] Checked all GitHub Actions related to this PR that can block the merge

## Important links:
1. [Simbiose Quality Guidelines](https://github.com/s1mbi0se/company/wiki/Quality-Topics#table-of-contents)
2. [Metadata Database Modeling](https://docs.google.com/spreadsheets/d/1Y8_k5xV86kTjgfjbi-PGsJcubUKhWlT2x8ecpnMlyR0/edit?usp=sharing)
3. [Simbiose's Corner Cases Guide](https://www.simbioseventures.com/documentos-e-guias/procedimentos-e-checklists/corner-cases-guide)

### CODE REVIEW

File: core/common/src/main/java/alluxio/wire/FileInfo.java [hunk 1]
```diff
@@ -115,7 +115,7 @@ public String getPath() {
   }
 
   /**
-   * @return the file UFS path
+   * @return the file UFS (Under File Storage) path
```
Comment (caiopavanelli) [inline core/common/src/main/java/alluxio/wire/FileInfo.java]:
```suggestion
   * @return the file UFS (Under File System) path
```

File: core/common/src/main/java/alluxio/wire/FileInfo.java [hunk 2]
```diff
@@ -386,8 +392,13 @@ public FileInfo setName(String name) {
   }
 
   /**
-   * @param path the file path to use
-   * @return the file information
+   * Sets the path for this file information.
+   *
+   * @param path the new file path to use for the
+   *             file this object holds information
```
Comment (caiopavanelli) [inline core/common/src/main/java/alluxio/wire/FileInfo.java]: wrong aligment

File: core/common/src/main/java/alluxio/wire/FileInfo.java [hunk 3]
```diff
@@ -396,8 +407,11 @@ public FileInfo setPath(String path) {
   }
 
   /**
-   * @param ufsPath the file UFS path to use
-   * @return the file information
+   * Sets the Under File Storage (UFS) path for the file this object holds information from
```
Comment (caiopavanelli) [inline core/common/src/main/java/alluxio/wire/FileInfo.java]: - UFS means Under File System;
- Could simplify the phrase to be similar to `setPath`:
  - "path for this file information."

**Review by caiopavanelli — CHANGES_REQUESTED** [review]

File: core/common/src/main/java/alluxio/wire/FileInfo.java [hunk 4]
```diff
@@ -396,8 +407,11 @@ public FileInfo setPath(String path) {
   }
 
   /**
-   * @param ufsPath the file UFS path to use
-   * @return the file information
+   * Sets the path for this file information.
```
Comment (caiopavanelli) [inline core/common/src/main/java/alluxio/wire/FileInfo.java]:
```suggestion
   * Sets the UFS path for this file information.
```

**Review by caiopavanelli — CHANGES_REQUESTED** [review]
The request was interpreted literally, you should have maintained the first part of the sentence.

**Review by caiopavanelli — APPROVED** [review]

File: core/common/src/main/java/alluxio/wire/Configuration.java [hunk 5]
```diff
@@ -43,7 +43,8 @@
   private final String mPathConfHash;
 
   /**
-   * @return new configuration builder
+   * @return a new {@link Builder} for cluster- and path-level
```
Comment (jpedroantunes) [inline core/common/src/main/java/alluxio/wire/Configuration.java:46]: Please. define a summary for the javadoc (no problem if it is a little bit redundant)

File: core/common/src/main/java/alluxio/wire/FileInfo.java [hunk 6]
```diff
@@ -115,7 +115,7 @@ public String getPath() {
   }
 
   /**
-   * @return the file UFS path
+   * @return the file UFS (Under File System) path
```
Comment (jpedroantunes) [inline core/common/src/main/java/alluxio/wire/FileInfo.java:118]: Please. define a summary for the javadoc (no problem if it is a little bit redundant)

**Review by jpedroantunes — CHANGES_REQUESTED** [review]

File: core/common/src/main/java/alluxio/wire/Configuration.java (hunk 5 above)
Comment (abenaru-zz) [reply core/common/src/main/java/alluxio/wire/Configuration.java:46]: According to the [new Alluxio convention](https://github.com/Alluxio/alluxio/pull/11797/files), getters and setters should omit the method description if it is redundant and only use `@param` and `@return` descriptions. This is recent.

File: core/common/src/main/java/alluxio/wire/FileInfo.java (hunk 6 above)
Comment (abenaru-zz) [reply core/common/src/main/java/alluxio/wire/FileInfo.java:118]: According to the [new Alluxio convention](https://github.com/Alluxio/alluxio/pull/11797/files), getters and setters should omit the method description if it is redundant and only use `@param` and `@return` descriptions. This is recent.

**Review by jpedroantunes — APPROVED** [review]
